xml/System.Net.Http/HttpClientHandler.xml

```diff
@@ -506,7 +506,7 @@ If the <xref:System.Net.Http.HttpClientHandler.UseDefaultCredentials> property h
 
  The <xref:System.Net.Http.HttpClientHandler.DangerousAcceptAnyServerCertificateValidator%2A> property addresses this limitation. The delegate returned by the <xref:System.Net.Http.HttpClientHandler.DangerousAcceptAnyServerCertificateValidator%2A> property can be assigned to the <xref:System.Net.Http.HttpClientHandler.ServerCertificateCustomValidationCallback%2A> property, as the following example does:
 
-```cs
+ ```csharp
```

Comment:
````suggestion
```csharp
````

I take it the difference between cs/c#/csharp here is just consistency, or might it impact how things are rendered?

Reply:
`cs` and `c#` are invalid and don't render properly. Compare [this "cs" example](https://learn.microsoft.com/en-us/dotnet/api/system.net.http.httpclienthandler.dangerousacceptanyservercertificatevalidator?view=net-9.0#remarks) to [this "csharp" example](https://learn.microsoft.com/en-us/dotnet/api/system.net.http.headers.authenticationheadervalue.-ctor?view=net-9.0#system-net-http-headers-authenticationheadervalue-ctor(system-string-system-string)).